gtsam/nonlinear/nonlinear.i

```diff
@@ -522,6 +522,14 @@ virtual class DoglegParams : gtsam::NonlinearOptimizerParams {
   void setVerbosityDL(string verbosityDL) const;
 };
 
+#include <gtsam/nonlinear/GncParams.h>
+template<class BaseOptimizerParameters>
```

Comment:
You shouldn't be specifying templates as `template<class T>`, it should be just `template<T>`.
Same deal for `GncOptimizer`.

Reply:
Got it, thank you Varun